src/utils/getFilteredPeople.tsx

```diff
@@ -0,0 +1,57 @@
+import { Person } from '../types';
+import { FilterParams } from '../types/FilterParams';
+import { Order } from '../types/Order';
+import { SortType } from '../types/SortType';
+
+export const getFilteredPeople = (people: Person[], {
+  query,
+  centuries,
+  sex,
+  sort,
+  order,
+}: FilterParams): Person[] => {
+  let filteredPeople = [...people];
```

Comment:
The `filter` array method returns a copy of the array, so you don't need to do it manually